packages/miniflare/test/plugins/browser/index.spec.ts

```diff
@@ -209,11 +210,11 @@ export default {
 `;
 
 const isWindows = process.platform === "win32";
-(isWindows ? test.skip : test.serial)(
+(isWindows ? test.skip : test)(
 	"fails if browser session already in use",
-	async (t) => {
+	async () => {
 		// We set the timeout quite high here as one of these tests will need to download the Chrome headless browser
-		t.timeout(20_000);
+		// Vitest handles timeouts via test options
```

Comment:
These timeouts can be provided in a TestOptions object that is passed to the `test` function. E.g. `test("...", { timeout: 20_000 }, async () => { ... });`